aws/atlantis/providers.tf

```diff
@@ -1,4 +1,5 @@
 terraform {
+  required_version = ">= 1.1.8"
```

Comment:
@stylianosrigas is it possible to make this variable? If not I am fine as it is.

Reply:
@stafot unfortunately you cannot put a variable in this section